adoc/MAIN.release-notes.adoc

```diff
@@ -37,7 +37,7 @@ kubectl -f delete addons/gangway/gangway.yaml
 * Delete the original image from node where gangway is running
 +
 ====
-sudo crictl rmi <gangway-image-id>
+sudo crictl rmi GANGWAY_IMAGE_ID
```

Comment:
Will this be replaced?

Reply:
Who can provide the ID?

Reply:
registry.suse.com/caasp/v4/gangway:3.1.0